rpc_jobs/defaults.yml

```diff
@@ -3,7 +3,7 @@
     name: global
     IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
     FLAVOR: "performance2-15"
-    REGION: "DFW"
+    REGION: "DFW, ORD"
```

Comment:
I think you'd need to make `all_regions` a parameter also because we can only build onmetal v1 in IAD.  For the mnaio jobs, you don't want it falling back to the other regions currently defined in `all_regions`.